src/test/java/com/ibm/cloudant/kafka/connect/CloudantSourceTaskTest.java

```diff
@@ -60,7 +63,8 @@ protected void setUp() throws Exception {
         data = new JSONArray(tokener);
 
         // Load data into the source database (create if it does not exist)
-        JavaCloudantUtil.batchWrite(sourceProperties, data);
+        JavaCloudantUtil.batchWrite(sourceProperties,
```

Comment:
There are a few instances of this awkward mapping from `org.json` in test code - I've tried to make the changes as minimal as possible but would love to get rid of that library altogether at a later date.